packages/core/src/app.ts

```diff
@@ -2073,7 +2073,27 @@ export class App {
     }
 
     getAccountFeatures() {
-        return this.getAccountProvisioning()?.features || new AccountFeatures();
+        const accountFeatures = this.getAccountProvisioning()?.features || new AccountFeatures();
+
+        // If there's a "premium feature" disabled, try to get an org's features instead
+        if (this._hasPremiumFeaturesDisabled(accountFeatures)) {
+            for (const orgId of this.getAccountProvisioning()?.orgs) {
+                const org = this.getOrg(orgId);
+                if (org) {
+                    const orgFeatures = this.getOrgFeatures(org.info);
+                    if (!this._hasPremiumFeaturesDisabled(orgFeatures)) {
+                        accountFeatures.attachments = orgFeatures.attachments;
+                        accountFeatures.securityReport = orgFeatures.securityReport;
+                        accountFeatures.totpField = orgFeatures.totpField;
+                        accountFeatures.notesField = orgFeatures.notesField;
+                        accountFeatures.itemHistory = orgFeatures.itemHistory;
+                        return accountFeatures;
```

Comment:
I intentionally return without saving here so that if the user leaves the org, their account won't "magically" keep the features enabled.